asset/js/setup.js

```diff
@@ -28,6 +40,9 @@ function setupAnchors() {
                          () => jQuery(heading).find('.fa.fa-anchor').css('visibility', 'hidden'));
     }
   });
+
+  setupDummySpans();
```

Comment:
Adding dummy spans is an interesting approach, but it does feel a little hackish. I wonder if we can make changes to the `scrollToUrlAnchorHeading` function to update the scrolling behaviour to do this in a cleaner way? 

If this is something that you've already tried, or if you feel otherwise about that approach, feel free to bounce this back to me.

Reply:
![image](https://user-images.githubusercontent.com/28982486/72664025-1b5cd300-3a34-11ea-8451-a062a9ce846a.png)

I think adding a dummy node is quite common in terms of anchoring... Above is the example in Docusarus, they also adopted this approach (only difference is they use <a> instead of <span> and they put the anchor inside of <h>).

> Adding dummy spans is an interesting approach, but it does feel a little hackish. I wonder if we can make changes to the scrollToUrlAnchorHeading function to update the scrolling behaviour to do this in a cleaner way?

If I understand it correctly, isn't `scrollToUrl..` only called once after the Vue instance is mounted?

Reply:
That makes sense, thanks for the explanation :)